---
int64 leftTicks
int64 rightTicks